src/App.jsx

```diff
@@ -11,7 +11,7 @@ function App() {
         <h1>Countries Stats</h1>
       </header>
       <Routes>
-        <Route path="/" element={<MainPage />} />
+        <Route path="/countries-database/" element={<MainPage />} />
```

Comment:
[OPTIONAL]
- [ ] Can you please make the homepage the default route? On opening the app, the default route is empty because you're not making any of your components the default route.